backend/app/Http/Controllers/ChoiceController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Choice;
+
+class ChoiceController extends Controller
+{
+    public function index()
```

Comment:
Where do you want to apply this? It will just show all choices and I cant understand why we are retrieving all choices

Reply:
Yes sir I was not using it and I will just delete it.